codegen/src/main/java/software/amazon/awssdk/codegen/poet/client/specs/Ec2ProtocolSpec.java

```diff
@@ -65,19 +65,19 @@ private MethodSpec dryRunMethod() {
                         "null",
                         "responseHandler",
                         "dryRunRequest")
-                .addStatement("throw new $T($S)", AmazonClientException.class,
+                .addStatement("throw new $T($S)", SdkClientException.class,
                         "Unrecognized service response for the dry-run request.")
                 .endControlFlow()
-                .beginControlFlow("catch (AmazonServiceException ase)")
-                .beginControlFlow("if (ase.getErrorCode().equals($S) && ase.getStatusCode() == 412)",
+                .beginControlFlow("catch (SdkServiceException exception)")
+                .beginControlFlow("if (exception.errorCode().equals($S) && ase.statusCode() == 412)",
```

Comment:
ase.statusCode() -> exception.statusCode()
ase has been renamed to exception.

Similarly in line 76